locale/foreman.pot

```diff
@@ -14994,38 +15037,69 @@ msgstr ""
 msgid "Review before build"
 msgstr ""
 
+#:
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/BuildModal.js:68
+msgid "Build enables host {hostName} to rebuild on next boot"
```

Comment:
This looks like the new js translations are using a different way of marking variables for interpolations, I'm not sure if this will work properly with the whole translation process? Previously, `%s` was used to mark a single interpolation, and `%{whatever}` to mark named interpolations. There are also checks that can be run with `pofilter` to make sure the translations are correct (e.g. same number of variables in source and translated string), and they may not work properly with these.

Reply:
The hostName shouldn't be translated and is an html object